### ISSUE TITLE
How to reproduce semantic segmentation?

### ISSUE THREAD

**Issue by jianlong-yuan** [zdaxie/PixPro#2]
Can you release the downstream task of semantic segmentation code?

**Comment by zdaxie** [comment]
Thanks for your attention. We are busy lately and we will consider releasing it in the future.

**Comment by jianlong-yuan** [comment]
Can you provide the parameter setting of segmentation? eg. learning rate, batch size, weight decay?

**Comment by code6levels** [comment]
I hope to know how to use this model  on  weakly-supervised semantic segmentation
